modules/migrations/onedev.go

```diff
@@ -0,0 +1,562 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
```

Comment:
is there a **public onedev** instance we could create a org *gitea* and put a **test repo to migrate** like we did with github and gitlab?

-> tld: `opendev_test.go` ;)

Reply:
There is only https://code.onedev.io/. I think it is like https://gitea.com, only for internal use. Without an instance I could only mock the api calls for tests.

Reply:
@KN4CK3R now we have: https://code.onedev.io/projects/go-gitea-test_repo